### PR TITLE
Serhii2

### PR DESCRIPTION
**Pull request by serhiylev**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -17,10 +17,13 @@ import {ChatComponent} from './chat/chat.component';
...
+import {MyProfileComponent} from "./my-profile/my-profile.component";
+import {AdminApproveStartComponent} from "./admin-approve-page/admin-approve-start/admin-approve-start.component";
+import {ServiceProviderViewComponent} from "./admin-approve-page/service-provider-view/service-provider-view.component";
+import {CustomerViewComponent} from "./admin-approve-page/customer-view/customer-view.component";
+import {FavoriteComponent} from "./favorite/favorite.component";
 import {PortfolioComponent} from './portfolio/portfolio.component';
 import {EditPostComponent} from './portfolio/edit-post/edit-post.component';
-import {AddPostComponent} from './portfolio/add-post/add-post.component';
-import {MyProfileComponent} from './my-profile/my-profile.component';
 import {NotFoundComponent} from './not-found/not-found.component';
```
Comment (AndriY2389) [inline src/app/app-routing.module.ts]: import {AddPostComponent} from './portfolio/add-post/add-post.component';  you delete this one - is it OK ?

File: src/app/admin-approve-page/customer-view/customer-view.component.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<app-menu-bar [showForCustomers]="true"></app-menu-bar>
+
+<div class="container-fluid">
+<div class="row rounded border border-dark" style="margin: 20px; padding: 20px" *ngFor="let customer of customers">
+  <div class="col-lg-2">
+    <img class="img_styles rounded" alt="{{customer.userDTO.image}}" width="100" height="100"
+         src="http://localhost:8080/customers/image/{{customer.userDTO.image}}"/>
+  </div>
```
Comment (DimaNarepeha) [inline src/app/admin-approve-page/customer-view/customer-view.component.html:8]: Localhost will not work on heroku, man

**Review by DimaNarepeha — CHANGES_REQUESTED** [review]

File: src/app/admin-approve-page/service-provider-view/service-provider-view.component.ts [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (chen0899) [inline src/app/admin-approve-page/service-provider-view/service-provider-view.component.ts]: ????

File: src/app/favorite/favorite.component.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+      <div class="blog_left_sidebar" *ngFor="let serviceProvider of serviceProviders ">
+        <article class="blog_style1">
+          <div class="blog_text">
+            <div class="blog_text_inner" style="margin-bottom: 13%;">
+              <div class="cat">
+                <a><i class="fas fa-map-marker-alt" aria-hidden="true"></i> {{serviceProvider.location.city}}</a>
+                <a><i class="far fa-calendar-alt" aria-hidden="true"></i> {{serviceProvider.registrationDate}}
+                </a>
+                <a href="#"><i class="far fa-comments" aria-hidden="true"></i> {{serviceProvider.countComment}}</a>
```
Comment (chen0899) [inline src/app/favorite/favorite.component.html:22]: router link, not href

**Pull request closed without merging**